cluster/utils_windows.go

```diff
@@ -0,0 +1,15 @@
+// +build !linux,!darwin
+
+package cluster
+
+import (
+	"errors"
+	"net"
+	"time"
+)
+
+// SetTCPUserTimeout doesn't support Windows because Go doesn't support
+// the option and swarm doesn't support cgo
+func setTCPUserTimeout(conn *net.TCPConn, uto time.Duration) error {
+	return errors.New("Go doesn't have native support for TCP_USER_TIMEOUT for this platform")
```

Comment:
Thanks for bringing it up. TCP_USER_TIMEOUT is a good to have feature. Dockerclient doesn't fail if this setting is not supported by a system. I  will add a comment for this.